### ISSUE TITLE
[FR] compliance with RFC draft-ietf-dnsop-rfc7958bis-06

### ISSUE THREAD

**Issue by lbaumbauer** [NLnetLabs/unbound#1173]
**Describe the desired feature**
Compliance with RFC draft-ietf-dnsop-rfc7958bis-06

**Potential use-case**
Update the trust anchor file to the latest version with auto-trust-anchor-file and use it in unbound.

Further reading
https://lists.dns-oarc.net/pipermail/dns-operations/2024-November/022711.html
https://datatracker.ietf.org/doc/draft-ietf-dnsop-rfc7958bis/

**Comment by wcawijngaards** [comment]
Unbound already supports this. Unbound has been tested with the new trust anchor publication format from the draft. This works, it turns out, without modifications needed. The xml format has been extended but that did not need code modification for the handling code. Or was there a different reason for the issue ticket?

In unbound, the unbound-anchor tool is used to perform the trust anchor fetch using this mechanism. The auto-trust-anchor-file unbound.conf option uses RFC5011 for tracking the key, that likely works so well that the unbound-anchor tool is mostly not needed for operations, but in some corner cases, specifically when the server was not online for the period, the unbound-anchor tool exists. The unbound-anchor program is documented in https://unbound.docs.nlnetlabs.nl/en/latest/manpages/unbound-anchor.html . It can be used before starting the server to update the trust anchor file, by setting it to run just before the server starts. Or it can be run from the commandline.

**Comment by lbaumbauer** [comment]
Thank you for the clarification, this is all i needed to know.
